integration/spark/src/main/scala/org/apache/carbondata/api/CarbonStore.scala

```diff
@@ -19,13 +19,13 @@ package org.apache.carbondata.api
 
 import java.io.{DataInputStream, FileNotFoundException, InputStreamReader}
 import java.time.{Duration, Instant}
-import java.util
 import java.util.{Collections, Comparator}
 
 import scala.collection.JavaConverters._
 import scala.util.control.Breaks.{break, breakable}
 
 import com.google.gson.Gson
+import java.util
```

Comment:
why this change? I think CI will fail because of this change

Reply:
reverted